Test/Recurly.Test.csproj

```diff
@@ -55,84 +44,21 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="FluentAssertions, Version=4.18.0.0, Culture=neutral, PublicKeyToken=33f2691a05b67b6a, processorArchitecture=MSIL">
-      <HintPath>..\packages\FluentAssertions.4.18.0\lib\net45\FluentAssertions.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="FluentAssertions.Core, Version=4.18.0.0, Culture=neutral, PublicKeyToken=33f2691a05b67b6a, processorArchitecture=MSIL">
-      <HintPath>..\packages\FluentAssertions.4.18.0\lib\net45\FluentAssertions.Core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.configuration" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
```

Comment:
Dont you still need these?  I dont see them added anywhere else.

Reply:
Since the Test project pulls in the main project, I thought these weren't needed anymore. The tests all run/pass with this configuration, though I suppose that's just in the container.. 🤔

Reply:
ah ok, that may be true.  Only one way to find out tho.  I think ill need to open this on a windows machine in VS

Reply:
yeap all good here!